### PR TITLE
Edits from customer comments, issues 173 and 273

### PR DESCRIPTION
**Pull request by kraigb**
Plus some linting cleanup

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/Consume-Packages/Configuring-NuGet-Behavior.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Consume-Packages/Configuring-NuGet-Behavior.md) | :white_check_mark:Succeeded | |
[docs/Create-Packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Create-Packages/Creating-a-Package.md) | :white_check_mark:Succeeded | |
[docs/Reference/extensibility/NuGet-Credential-Providers-for-Visual-Studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Reference/extensibility/NuGet-Credential-Providers-for-Visual-Studio.md) | :white_check_mark:Succeeded | |
[docs/Reference/extensibility/nuget-exe-Credential-Providers.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Reference/extensibility/nuget-exe-Credential-Providers.md) | :white_check_mark:Succeeded | |
[docs/Schema/Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Schema/Target-Frameworks.md) | :white_check_mark:Succeeded | |
[docs/Tools/cli-ref-add.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Tools/cli-ref-add.md) | :white_check_mark:Succeeded | |
[docs/Tools/cli-ref-config.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Tools/cli-ref-config.md) | :white_check_mark:Succeeded | |
[docs/Tools/cli-ref-delete.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Tools/cli-ref-delete.md) | :white_check_mark:Succeeded | |
[docs/Tools/cli-ref-environment-variables.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Tools/cli-ref-environment-variables.md) | :white_check_mark:Succeeded | |
[docs/Tools/cli-ref-install.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Tools/cli-ref-install.md) | :white_check_mark:Succeeded | |
[docs/Tools/cli-ref-locals.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Tools/cli-ref-locals.md) | :white_check_mark:Succeeded | |
[docs/Tools/cli-ref-push.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Tools/cli-ref-push.md) | :white_check_mark:Succeeded | |
[docs/Tools/cli-ref-restore.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Tools/cli-ref-restore.md) | :white_check_mark:Succeeded | |
[docs/Tools/cli-ref-update.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Tools/cli-ref-update.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C12%5C12%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201712122222340168-608%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fwacmDfLw2c2l5Q6O7Iof90sjybGw7en3%2BkwqPraj5k%3D&st=2017-12-12T22%3A18%3A25Z&se=2018-01-12T22%3A23%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/Tools/cli-ref-restore.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ where `<projectPath>` specifies the location of a solution, a `packages.config`
 | Recursive | *(4.0+)* Restores all references projects for UWP and .NET Core projects. Does not apply to projects using `packages.config`. |
 | RequireConsent | Verifies that restoring packages is enabled before downloading and installing the packages. For details, see [Package Restore](../consume-packages/package-restore.md). |
 | SolutionDirectory | Specifies the solution folder. Not valid when restoring packages for a solution. |
-| Source | Specifies list of package sources (as URLS) to use for the restore. |
+| Source | Specifies list of package sources (as URLS) to use for the restore. If omitted, the command uses the sources provided in configuration files, see [Configuring NuGet behvior](../Consume-Packages/Configuring-NuGet-Behavior.md). |
```
Comment (xavierdecoster) [inline docs/Tools/cli-ref-restore.md]: typo `behvior`

File: docs/Tools/cli-ref-push.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ where `<packagePath>` identifies the package to push to the server.
 | Help | Displays help information for the command. |
 | NonInteractive | Suppresses prompts for user input or confirmations. |
 | NoSymbols | *(3.5+)* If a symbols package exists, it will not be pushed to a symbol server. |
-| Source | Specifies the server URL. With NuGet 2.5+, NuGet will identify a UNC or local folder source and simply copy the file there instead of pushing it using HTTP.  Also, starting with NuGet 3.4.2, this is a mandatory parameter unless the `NuGet.Config` file specifies a *DefaultPushSource* value. |
+| Source | Specifies the server URL. With NuGet 2.5+, NuGet will identify a UNC or local folder source and simply copy the file there instead of pushing it using HTTP.  Also, starting with NuGet 3.4.2, this is a mandatory parameter unless the `NuGet.Config` file specifies a *DefaultPushSource* value (see [Configuring NuGet behvior](../Consume-Packages/Configuring-NuGet-Behavior.md)). |
```
Comment (xavierdecoster) [inline docs/Tools/cli-ref-push.md]: typo `behvior`

File: docs/Tools/cli-ref-update.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ where `<configPath>` identifies either a `packages.config` or solution file that
 | RepositoryPath | Specifies the local folder where packages are installed. |
 | Safe | Specifies that only updates with the highest version available within the same major and minor version as the installed package will be installed. |
 | Self | *(1.4+)* Updates nuget.exe to the latest version; all other arguments are ignored. |
-| Source | Specifies a list of package sources to use for the updates. |
+| Source | Specifies list of package sources (as URLS) to use for the updates. If omitted, the command uses the sources provided in configuration files, see [Configuring NuGet behvior](../Consume-Packages/Configuring-NuGet-Behavior.md). |
```
Comment (xavierdecoster) [inline docs/Tools/cli-ref-update.md]: typo `behvior`

Comment (xavierdecoster) [inline docs/Tools/cli-ref-update.md]: should the `s` be lowercased in `URLS`?

**Comment by kraigb** [comment]
@xavierdecoster Thanks for catching those glitches. I've fixed in https://github.com/NuGet/docs.microsoft.com-nuget/pull/612.

**Pull request merged**